.circleci/config.yml

```
@@ -409,6 +409,20 @@ jobs:
       - store_test_results:
           path: e2e-tests/visual-regression/cypress/results
 
+  e2e_tests_contentful:
+    <<: *e2e-executor
+    environment:
+    CYPRESS_PROJECT_ID: 2193cm
+    CYPRESS_RECORD_KEY: 57e9563e-af49-494c-837c-5af53c2d6f76
```

I think this is just whitespace indentation problem:
```suggestion
      CYPRESS_PROJECT_ID: 2193cm
      CYPRESS_RECORD_KEY: 57e9563e-af49-494c-837c-5af53c2d6f76
```